### DISCUSSION TITLE
Compatibility problems with the latest stable version

### DISCUSSION THREAD

**Discussion by fmaestre98** [isar/isar#1609] [Help]
Hello, could you tell me if a stable version will be released soon with which I can avoid this type of problems "isar 3.1.0+1 depends on js ^0.6.4 and xxxxxxxxxxx depends on js ^0.7.1, xxxxxxxx from path is incompatible with xxxxxxxxx"
Thank you

**Comment by vicenterusso** [comment]
Check [Isar Community](https://github.com/isar-community/isar) fork
